test/CassandraStatements.js

```diff
@@ -0,0 +1,46 @@
+const chai = require('chai');
+const assert = chai.assert;
+const cassandraStatements = require('../src/statements/CassandraStatements');
+
+describe('Tests for CassandraStatements.js', function() {
+
+  describe('#prepareInsertTopic(topic)', function() {
```

Comment:
The code that this tests is trivial: just returning a constant string and unpacking an object into an array. As such, this is the type of test that doesn't add much value since it doesn't encode anything really valuable to the system under test.

Having too many tests is just as bad as having too few tests since more tests increases the cost of code changes/refactors and gives a false sense of security. Ideally, we should test at system/interface boundaries.

For Cassandra, a meaningful test would be: "given these inputs, does this resolver hit the DB with the correct query?" or "given this DB response, does the resolver produce the correct output?". Both of these tests cover scenarios, not implementations so if they pass, we can assume with some degree of confidence that the resolver works. Additionally, the tests will likely not have to change if the implementation of the resolver changes.

Basically: it's better to test contracts, not implementations.